ui/desktop/app/routes/scopes/scope/projects/targets/index.js

```diff
@@ -52,6 +58,24 @@ export default class ScopesScopeProjectsTargetsIndexRoute extends Route {
       this.can.can('connect target', target),
     );
 
-    return { targets: targets, totalItems };
+    let targetsWithSessions = [];
+    for (const target of targets) {
+      let sessions = await this.store.query('session', {
+        query: {
+          filters: {
+            target_id: { equals: target.id },
+            status: [
+              { equals: STATUS_SESSION_ACTIVE },
+              { equals: STATUS_SESSION_PENDING },
+              { equals: STATUS_SESSION_CANCELING },
```

Comment:
Just realized I forgot to support for not equals, should I add it so you could do just `{ notEquals: STATUS_SESSION_TERMINATED }`?

Reply:
oh yes plz, that would make this a lot neater.

Reply:
Either way I added [support](https://github.com/hashicorp/boundary-ui/pull/2029) since we should have it regardless haha